card/mod/01_core/chunk/include.rb

```diff
@@ -5,7 +5,7 @@
 module Card::Chunk
   class Include < Reference
     cattr_reader :options
-    @@options = ::Set.new [ :inc_name, :inc_syntax, :view, :items, :type, :size, :title, :hide, :show, :structure ]
+    @@options = ::Set.new [ :inc_name, :inc_syntax, :view, :items, :type, :size, :title, :hide, :show, :structure, :unlabeled ]
```

Comment:
A case of too much respect for existing code :)  The label used to be handled with args so I needed that but makes sense to turn that into an optional view